src/main/webapp/forum.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Seniors Connect</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <div class="navbar">
+            <ul>
+                <li><a href="index.html">Seniors Connect</a></li>
```

Comment:
Is this needed as a duplicate for Home?

Reply:
@rmr167

Reply:
I commented this out from the navbar, I think we had a discussion that "Seniors Connect" (&logo) will be in the upper left hand corner, and the right hand corner will have the navbar. I believe clicking the "Seniors Connect" in the upper left hand corner users will be redirected to the home page and so will the "Home" in the nav bar.

Reply:
That makes sense. Can you add a TODO to indicate that? Something like:

<!-- TODO: Show this at top left corner. -->

Reply:
Yeah that is what I was referring to.  Praneet, should I change this or should you since it is your PR?

Reply:
I changed it in my PR